app/adapters/whats_app_adapter/twilio_outbound.rb

```diff
@@ -54,10 +54,9 @@ def send_resubscribe_error_message!(contributor, organization)
 
       def send_message_template!(recipient, message)
         recipient.update(whats_app_message_template_sent_at: Time.current)
-        text = I18n.t("adapter.whats_app.request_template.new_request_#{time_of_day}_#{rand(1..3)}", first_name: recipient.first_name,
-                                                                                                     request_title: message.request.title)
-        WhatsAppAdapter::TwilioOutbound::Text.perform_later(organization_id: message.organization.id, contributor_id: recipient.id,
```

Comment:
Is it possible that we forgot to delete the file defininig `WhatsAppAdapter::TwilioOutbound::Text`? :thinking:

Reply:
We didn't delete this class because it is still used to send freeform messages to contributors. If they respond to a template for example then it opens a 24-hour window where we can send them whatever message we want. This is how they receive the actual personalized message that is created for them when a request is created by a user of the app.

One thing is that the welcome message is a template and we do not account for this in this PR because the idea is to prevent loss of ongoing service, but not to support new contributors onboarding.